src/Items/ItemFishingRod.h

```diff
@@ -104,22 +104,27 @@ class cItemFishingRodHandler :
 				int ItemCategory = Random.RandInt(99);
 				if (ItemCategory <= 4)  // Treasures 5%
 				{
-					int Treasure = Random.RandInt(5);
-					switch (Treasure)
+					switch (Random.RandInt(5))  // Each piece of treasure has an equal chance of 1 / 6
 					{
 						case 0:
 						{
-							Drops.Add(cItem(E_ITEM_BOW));  // TODO: Enchantments
+							cItem Bow = cItem(E_ITEM_BOW, 1, Random.RandInt<short>(50));
```

Comment:
Why are you constructing the items like this?

Reply:
Is there another/better way to do this?

Reply:
What's wrong with the normal way?
```cpp
cItem Bow(E_ITEM_BOW, 1, Random.RandInt<short>(50));
```